examples/awq/README.md

```diff
@@ -10,8 +10,7 @@ The AWQ recipe has been inferfaced as follows, where the `AWQModifier` adjusts m
 
 ```python
 recipe = [
-    AWQModifier(bits=4, symmetric=False),
-    QuantizationModifier(
+    AWQModifier(
```

Comment:
want to use the scheme here?

Reply:
We could, I can in a future PR, I want to go through and remove `calib_data` from oneshot/session etc. I can tackle it then to unblock this error
\

Reply:
we discussed that the other PR is not needed, as other modifiers rely on `calib_data`. so i updated on this PR